kodilla-rps/src/main/java/com/kodilla/rps/RpsRunner.java

```diff
@@ -46,20 +52,21 @@ public static void main(String[] args){
                 gameCount = sc.nextInt();
                 continue;
             }
+            //wydzielic do comupterRound
             //Generating computer move
 
             Random random = new Random();
             int computerMove = random.nextInt(3);
 
             //actual game
 
-
+//wydzielic do metody play 
             Game game = new Game(move,computerMove);
             System.out.println(game.playerMoveTranslate(move) + " vs " + game.computerMoveTranslate(computerMove));
```

Comment:
println(game.getPlayerMove() vs game.getComputerMove() ), ale może lepiej niech klasa game to wypisuje